src/resources/Global/OrganizationMonitoring/OrganizationMonitoring.ts

```diff
@@ -0,0 +1,12 @@
+import Resource from '../../Resource.js';
+import {OrganizationMonitoringEndpointUsageModel} from './OrganizationMonitoringInterfaces.js';
+
+export default class OrganizationEndpointUsage extends Resource {
+    static baseUrl = '/rest/monitoring/global/organizations';
```

Comment:
The OrganizationConfiguration class uses the same baseUrl, would it make sense to move the monitoring stuff in there instead of creating a separate class of its own?